SofaKernel/SofaBase/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(SOFABASE_TARGETS
 
 foreach(TARGET ${SOFABASE_TARGETS})
     add_subdirectory(../modules/${TARGET} ${CMAKE_CURRENT_BINARY_DIR}/${TARGET})
+    set_target_properties(${TARGET} PROPERTIES FOLDER SofaBase)
```

Comment:
this line is new compare to the SofaCommon pluginization, normal?

Reply:
It is a new line, because I am removing the macro (in the packaged modules) which automatically add the ${Target} in the IDE folder SofaBase.
This is purely a cosmetic line (only MSVC, maybe qt creator as well)